frontend/src/static/js/main.js

```diff
@@ -250,7 +250,11 @@ window.app = new window.Vue({
       return JSON.stringify(dataObj);
     },
 
-    getRepoSenseLink() {
+    getRepoSenseMainSiteLink() {
+      return 'https://github.com/reposense/RepoSense';
+    },
+
+    getRepoSenseVersionSpecificLink() {
```

Comment:
`Specific` can be removed cause its implied with singularity

Reply:
Okay.